docs/pages/why-web-components.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Why web components
+description: Explanation of why we use web components in the CA design system
+keywords: 
+---
+
+
+## Component-based development
+
+Building sites with components has a lot of benefits for site visitors and the teams that run them.
+
+### How components work
+
+Components are bite-sized sections of code that link together to create a website. They&rsquo;re kind of like LEGO bricks. They&rsquo;re often uniform and reusable.
+
+Components help developers and designers save time. They help you avoid rewriting redundant code. You can also easily plug them into and remove them from code using HTML tags.
+
+The California Design System uses the minimal amount of tech required for each component. This is&nbsp;often just HTML &amp; CSS. When we need JavaScript, we take advantage of the component model built into the browser. We use the custom elements part of the web components specification. This allows us to define new HTML tags like &lt;cagov-accordion&gt; or &lt;cagov-page-feedback&gt;, which are inert until upgraded by provided JavaScript.
+
+### Examples of our components
+
+The homepage also uses the <a href="https://designsystem.webstandards.ca.gov/components/feature-card/readme/">feature card</a>&nbsp;component. This component requires only HTML & CSS. 
+
+<img alt="" src="/content/img/screenshot-feature.jpg">
+
+In some cases, we can embed components in other components. For example, we put a button in the feature card.
+
+We use the <a href="/components/page-feedback/readme/">page feedback component</a> at the bottom of every page. This component requires HTML, CSS and javascript. It uses a custom element tag associated with a javascript class.
```

Comment:
JavaScript - https://docs.google.com/document/d/1254bcSpS5dofjn1jIg-0XawysJ2huoGd3vabkNS54iE/edit#heading=h.b0cyzsqu2fej

Reply:
Thank you

Reply:
updated